Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -144,13 +132,23 @@ public class NavigationCameraStateTransition: CameraStateTransition {
         if let animatorCenter = animatorCenter, animatorCenter.isRunning {
             animatorCenter.stopAnimation()
         }
-
-        let centerTimingParameters = UICubicTimingParameters(controlPoint1: CGPoint(x: 0.0, y: 0.0), controlPoint2: CGPoint(x: 1.0, y: 1.0))
-        animatorCenter = mapView.camera.makeAnimator(duration: duration, timingParameters: centerTimingParameters) { (transition) in
-            transition.center.toValue = center
+        
+        // Check whether the location change is larger than a certain threshold when current camera state is following.
+        var updateCameraCenter: Bool = true
+        if state == .following {
+            let metersPerPxiel = getMetersPerPixelAtLatitude(center.latitude, Double(zoom))
```

Comment:
Typo: pixel.

The latitude could differ between the original and target cameras, but at the moment I can’t think of a situation where that difference would be so great as to affect whether to avoid animating, yet so small as to cause the method to avoid animating. 👍